universe/config.json

```diff
@@ -103,7 +103,53 @@
                     "type": "string",
                     "description": "Specify the integer UID for the Linux user, overriding service.user, when running the Spark Driver and Executor containers with the Docker Engine. Directly translates to docker run '--user' parameter. Note: This should typically be set to 99 when running as nobody (default) on RHEL/CentOS.",
                     "default": ""
-                }
+                },
+		"tls": {
```

Comment:
Please use spaces for indentation here. Right now both tabs and spaces are used in one file.